### PR TITLE
Enable the hparams plugin

### PR DESCRIPTION
**Pull request by erzel**
* Motivation for features / changes: Enable the HParams plugin in Tensorboard. HParams allows visualizations of experiments ran for hyperparameter tuning.

* Screenshots of UI changes
image

* Detailed steps to verify changes work correctly (as executed by you)

```
bazel test //tensorboard/plugins/hparams/...:all
bazel run //tensorboard/plugins/hparams:hparams_demo 
bazel run //tensorboard -- --logdir /tmp/hparams_demo 
```

and (briefly) verified manually that the plugins works as expected.

### CODE REVIEW

**Comment by erzel** [comment]
@manivaradarajan @GalOshri 
Here is the PR for enabling hparams in TB. Please take a look, thanks.

 Erez.

File: tensorboard/BUILD [hunk 1]
```diff
@@ -91,8 +91,8 @@ py_test(
     deps = [
         ":manager",
         ":version",
-        "//tensorboard/util:tb_logging",
         "//tensorboard:expect_tensorflow_installed",
+        "//tensorboard/util:tb_logging",
```
Comment (wchargin) [inline tensorboard/BUILD]: Please revert spurious changes to this file.

If we want to maintain the invariant that are files match Buildifier
style, we’ll add CI tests to make sure that continues to be the case.
Attempting to fix the style on an ongoing basis without backsliding is
hopeless and just pollutes the blame.

Comment (erzel) [reply tensorboard/BUILD]: Done.

**Comment by erzel** [comment]
Thanks @wchargin! PTAL.

**Comment by wchargin** [comment]
Thanks!

**Pull request merged**